docs/maps/map-series.md

```diff
@@ -15,7 +17,7 @@ For an overview of the `map` series options see the [API reference](https://api
        ...
     });
 
-<iframe style="width: 100%; height: 520px; border: none;" src="https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/maps/demo/category-map/" allow="fullscreen"></iframe>
+<iframe style="width: 100%; height: 520px; border: none;" src="https://www.highcharts.com/samples/embed/maps/demo/category-map" allow="fullscreen"></iframe>
```

Comment:
Why `www.` is added here but not in the demo above? Not sure which one is correct (probably both), but we can choose one and be consistent from now